Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml

```diff
@@ -0,0 +1,114 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SSLBL Feed
```

Comment:
Let's change the `id`, `name`, and `display` fields - `SSLBL` is not a good name IMO. Maybe `SSL Blacklist` or `SSL Abuse Blacklist`... I'm not sure. Whatever you decide, the filenames should be updated accordingly.